fms/utils/__init__.py

```diff
@@ -2,6 +2,7 @@
 from typing import Optional
 
 from torch.distributed import ProcessGroup
+from fms.utils import gptq
```

Comment:
remove this, as we don't want to import AutoGPTQ libraries by default in FMS (as that would require adding to requirements.txt, etc.)